.gitignore

```diff
@@ -1,3 +1,4 @@
 coverage
 node_modules
 dist
+.vscode
```

Comment:
nit: configure your global git config to ignore this: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html